edi_project_oca/views/project_task.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+
+    <record id="quick_create_task_form_inherit" model="ir.ui.view">
+        <field name="model">project.task</field>
+        <field name="inherit_id" ref="project.quick_create_task_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//group" position="inside">
+                <field name="edi_disable_auto" string="Disable EDI workflow" />
+            </xpath>
+        </field>
+    </record>
+
+    <record id="view_task_form2_inherit" model="ir.ui.view">
+        <field name="model">project.task</field>
+        <field name="inherit_id" ref="project.view_task_form2" />
+        <field name="arch" type="xml">
+            <xpath expr="//div[@name='button_box']" position="inside">
+                <button
+                    type="object"
+                    class="oe_stat_button"
+                    icon="fa-retweet"
+                    invisible="exchange_record_count == 0"
+                    name="action_view_edi_records"
+                >
+                    <div class="o_form_field o_stat_info">
+                        <span class="o_stat_value">
+                            <field name="exchange_record_count" />
+                        </span>
+                        <span class="o_stat_text">EDI</span>
+                    </div>
+                </button>
+            </xpath>
+            <xpath expr="//page[@name='extra_info']/group[last()]" position="inside">
+                <group name="edi_project_group" string="Electronic Data Interchange">
```

Comment:
```suggestion
                <group name="edi_project_group" string="EDI">
```
Q: wouldn't be better to have a specific page for EDI? 
Just asking...

Reply:
Yep, can do that